packages/paste-design-tokens/gulpfile.ts

```diff
@@ -29,7 +29,7 @@ theo.registerFormat('es6.js', es6TokenFormat);
 theo.registerFormat('common.d.ts', dTSTokenFormat);
 theo.registerFormat('es6.d.ts', dTSTokenFormat);
 theo.registerFormat('sketchpalette', sketchpaletteTokenFormat);
-theo.registerFormat('generic.js', genericCommonJsTokenFormat);
+theo.registerFormat('generic.js', genericJsTokenFormat);
```

Comment:
Question: If anyone is using this import directly from design tokens package, should we be more explicit in our change log for design tokens? Is a consumer going to need to change the babel, webpack, whatever configs? Is that technically a breaking change?

Reply:
Yeah can make an explicit note for that package in the changeset. Hard to say if this should be considered breaking. The Paste website for example uses that import and its Webpack configuration didn't need tweaking.